igzip_042/include/types.h

```diff
@@ -0,0 +1,46 @@
+/**********************************************************************

```

Comment:
Come to think of it given all of these types are now supported by modern compilers is this file even necessary?